index.js

```diff
@@ -258,6 +257,7 @@ async function terraform(terraformDirectory, args) {
   if (uploadResponse.failedItems.length > 0) {
     core.setFailed('Failed to upload artfiact');
   }
+  core.endGroup();
```

Comment:
Now the error on line 237 is reported inside the group as well, that one was put outside the group deliberately :wink: Which is desired to quickly see the error(s), unless it auto-expands groups that contain `setFailed()` nowadays.

So, the 2nd option is probably cleanest to implement: separate group for the changes/artifact stuff :grimacing:

Reply:
Moved changes to separate group with the failure check outside the group